### PR TITLE
Cleanup setup (also update references to license file)

### PR DESCRIPTION
**Pull request by cjchapman**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -23,171 +24,189 @@
...
-              "hintplot = afdko.Tools.SharedData.FDKScripts.ProofPDF:hintplot",
-              "waterfallplot = afdko.Tools.SharedData.FDKScripts.ProofPDF:waterfallplot",
-              "clean_afdko = afdko.Tools.SharedData.FDKScripts.FDKUtils:clean_afdko",
-              "check_afdko = afdko.Tools.SharedData.FDKScripts.FDKUtils:check_afdko",
+
+    if cur_system == "Darwin":
+        more_keywords = ['Operating System :: MacOS :: MacOS X']
+    elif cur_system == "Windows":
+        more_keywords = ['Operating System :: Microsoft :: Windows']
+        scripts = [
```
Comment (miguelsousa) [inline setup.py]: This can be reduced to something like
```python
scripts = ['{}.{}'.format(path, 'exe') for path in scripts]
```

File: setup.py [hunk 2]
```diff
@@ -23,171 +24,189 @@
...
+        long_description += changelog.read()
+
+    console_scripts = get_console_scripts()
+
+    setup(name="afdko",
+          version="2.6.25b2",
+          description="Adobe Font Development Kit for OpenType",
+          long_description=long_description,
+          url='https://github.com/adobe-type-tools/afdko',
+          author='Read Roberts and many other font engineers',
```
Comment (miguelsousa) [inline setup.py]: Change to "Adobe Type team & friends"

File: setup.py [hunk 3]
```diff
@@ -23,171 +24,189 @@
...
+          author_email='rroberts@adobe.com',
```
Comment (miguelsousa) [inline setup.py]: Change to `afdko@adobe.com`

File: setup.py [hunk 4]
```diff
@@ -23,171 +24,189 @@
...
     else:
         raise KeyError("Do not recognize target OS: %s" % sys.platform)
     platform_name = get_platform()
-    return binDir, platform_name, curSystem
+    return bin_dir, platform_name, cur_system
 
 
-def compile(pgkDir):
+def compile_package(pkg_dir):
     if 0:
```
Comment (miguelsousa) [inline setup.py]: I don't understand this `if`. In what circumstances will it happen?

File: setup.py [hunk 5]
```diff
@@ -23,171 +24,189 @@
...
-    'afdko/Tools/%s/tx' % binDir,
-    'afdko/Tools/%s/type1' % binDir,
-]
-
-if curSystem == "Darwin":
-    moreKeyWords = ['Operating System :: MacOS :: MacOS X']
-elif curSystem == "Windows":
-    moreKeyWords = ['Operating System :: Microsoft :: Windows']
+def get_console_scripts():
+    console_scripts = [
```
Comment (miguelsousa) [inline setup.py]: Remove the `return` from the bottom and prepend it here.

File: setup.py [hunk 6]
```diff
@@ -23,171 +24,189 @@
...
 
-def compile(pgkDir):
+def compile_package(pkg_dir):
     if 0:
         print("Skipping compile - hard coded.")
         return
-    binDir, platform_name, curSystem = getExecutableDir()
-    programsDir = os.path.join(pgkDir, "Tools", "Programs")
-    if binDir == 'osx':
+    bin_dir, platform_name, cur_system = get_executable_dir()
```
Comment (miguelsousa) [inline setup.py]: Change to
```python
bin_dir, _, _ = get_executable_dir()
```

File: setup.py [hunk 7]
```diff
@@ -23,171 +24,189 @@
...
+    cmd = None
+    if bin_dir == 'osx':
         cmd = "sh BuildAll.sh"
-    elif binDir == 'win':
+    elif bin_dir == 'win':
         cmd = "BuildAll.cmd"
-    elif binDir == 'linux':
+    elif bin_dir == 'linux':
         cmd = "sh BuildAllLinux.sh"
     curdir = os.getcwd()
```
Comment (miguelsousa) [inline setup.py]: Change to `cur_dir`

File: setup.py [hunk 8]
```diff
@@ -23,171 +24,189 @@
...
-        binDir = "osx"
+    cur_system = platform.system()
+    if cur_system == "Windows":
+        bin_dir = "win"
+    elif cur_system == "Linux":
+        bin_dir = "linux"
+    elif cur_system == "Darwin":
+        bin_dir = "osx"
     else:
         raise KeyError("Do not recognize target OS: %s" % sys.platform)
```
Comment (miguelsousa) [inline setup.py]: Please convert all of these `"%s"` to `.format()`

File: setup.py (hunk 4 above)
Comment (cjchapman) [reply setup.py]: That seems to be test/debug code. I'll remove it.

File: setup.py (hunk 6 above)
Comment (cjchapman) [reply setup.py]: I'm going to change this so `get_executable_dir()` only returns the executable dir, like it says in the name.

File: setup.py (hunk 5 above)
Comment (cjchapman) [reply setup.py]: I'm going to clean this up much more than that.

**Comment by cjchapman** [comment]
@miguelsousa I think I got everything. If it looks OK to you, please merge. Thanks.

File: setup.py [hunk 9]
```diff
@@ -23,171 +24,178 @@
...
         cmd = "BuildAll.cmd"
-    elif binDir == 'linux':
+    elif bin_dir == 'linux':
         cmd = "sh BuildAllLinux.sh"
-    curdir = os.getcwd()
-    subprocess.check_call(cmd, cwd=programsDir, shell=True)
-    os.chdir(curdir)
+    cur_dir = os.getcwd()
+    assert cmd, 'Unable to form command for this platform.'
+    subprocess.check_call(cmd, cwd=programs_dir, shell=True)
```
Comment (miguelsousa) [inline setup.py]: Should this call be wrapped in a try/except so that we can listen for CalledProcessError?

Comment (cjchapman) [reply setup.py]: OK, done.

**Pull request merged**